src/ui/badge.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
```

Comment:
I realized today that the designs on this part were outdated in Figma :( and the new designs are not ready yet.. so I'll keep this as it is until we can use or remove